js/views/PageNav.js

```diff
@@ -0,0 +1,67 @@
+import electron from 'electron';
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+
+const remote = electron.remote;
+
+export default class PageNav extends View {
```

Comment:
@morebrownies 

I guess that also may mean, that once the app is loaded, we never want to obscure the pageNav, which might be a good thing. Any loading process should allow the user to implicitly cancel it by navigating to a different page and it's up to us a devs to handle that case.